src/content/plugins/split-chunks-plugin.md

```diff
@@ -18,9 +19,9 @@ Since webpack v4, the `CommonsChunkPlugin` was removed in favor of `optimization
 
 ## Defaults
 
-Out of the box `SplitChunksPlugin` should work great for most users.
+Out of the box `SplitChunksPlugin` should work well for most users.
```

Comment:
feels like it didnt work great for most users and we had to change to to `well` 😕

Reply:
In this case, "well" reads as the more natural adverb in this sentence.